### ISSUE TITLE
import/extensions reporting empty extension when target file does not exist

### ISSUE THREAD

**Issue by gajus** [benmosher/eslint-plugin-import#295]
> import/extensions Unexpected use of file extension "" for "./../errors/scrape_error"

Subject code:

```js
import ScrapeError from './../errors/scrape_error';
```

Configuration:

```json
"import/extensions": [
    2,
    "never"
],
```

Version:

> eslint-plugin-import@1.6.1

**Comment by lo1tuma** [comment]
Does the file actually have an extension? Does it exist at all? I’ve experienced something similar when trying to import a file that doesn’t exist.

**Comment by gajus** [comment]
You are right. The file does not exist.

In that case, this certainly calls for a better error.

**Comment by benmosher** [comment]
Possibly a dupe of #271. I'm going to look at this right now. Likely resolution will be not reporting anything for `extensions` if a source path cannot be resolved.
